cmd/release-controller/http_changelog.go

```diff
@@ -18,6 +19,31 @@ var (
 	reRHCoSVersion = regexp.MustCompile(`\* Red Hat Enterprise Linux CoreOS ((\d)(\d+)\.[\w\.\-]+)\n`)
 )
 
+/*
+TODO: I tried to make this use the official definition from:
+			github.com/openshift/library-go/pkg/image/dockerv1client
+      Unfortunately, it causes an issue with the modules:
+			go: github.com/openshift/release-controller imports
+				github.com/openshift/library-go/pkg/image/dockerv1client imports
+				github.com/fsouza/go-dockerclient imports
+				github.com/docker/docker/opts imports
+				github.com/docker/docker/daemon/network imports
+				github.com/docker/docker/runconfig imports
+				github.com/docker/docker/pkg/sysinfo imports
+				github.com/opencontainers/runc/libcontainer/cgroups imports
+				github.com/opencontainers/runc/libcontainer/configs imports
+				github.com/Sirupsen/logrus: github.com/Sirupsen/logrus@v1.6.0: parsing go.mod:
+				module declares its path as: github.com/sirupsen/logrus
+					    but was required as: github.com/Sirupsen/logrus
+*/
+type dockerImageConfig struct {
+	Architecture string `json:"architecture"`
+}
+
+type releaseInfoConfig struct {
+	Config *dockerImageConfig `json:"config"`
+}
```

Comment:
@smarterclayton Is it worth trying to resolve the module problem for importing the **dockerv1client** definition, or is extracting just what I need OK?  This is why I chose to make these 2 structs private and flag the whole thing as a TODO.

Reply:
Just extracting what you need is fine.  I don't even think you need the TODO, just "// extract architecture directly from the response"